src/FirebirdSql.Data.FirebirdClient/Services/FbBackup.cs

```diff
@@ -32,7 +32,7 @@ public sealed class FbBackup : FbService
 	public int Factor { get; set; }
 	public string SkipData { get; set; }
 	public FbBackupFlags Options { get; set; }
-	public FbBackupRestoreStatistics Statistics { get; set; }
+	public FbBackupRestoreStatistics? Statistics { get; set; } = default(FbBackupRestoreStatistics);
```

Comment:
The `default...` is not needed.

Reply:
If I see this correctly, this would change the previous behavior.

Reply:
How? The `Statistics` was empty before, thus the `isc_spb_bkp_stat` was empty as well.

Reply:
I had considered that in one case `isc_spb_bkp_stat` is passed empty to the server and in the other case not at all.

Reply:
But the end result will be the same. Or am I missing something?

Reply:
By default, the parameter `isc_spb_bkp_stat` is no longer passed to the server. 
This changes the behavior of the Firebird 2.0 server. There are no more errors during backup 😀
If we assume that the other servers do not make a difference if the parameter is empty or not passed, then the end result is the same.